.github/workflows/ci-docs.yml

```diff
@@ -26,3 +27,16 @@ jobs:
           skip: .git,*/package.json,*/package-lock.json,*.lock,.github,.vscode,assets
           ignore_words_file: .codespellignore
           check_hidden: false
+
+  check_docs_grammar:
+    name: Check Docs Grammar
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Run grammar check
+        uses: awreck/proofread-action@v1
```

Comment:
This action doesn't work without licence and they don't have a way to specify the licence even if we purchase one
https://github.com/awreck/proofread-action/issues/32